src/components/ModalWithImage.js

```diff
@@ -19,7 +19,6 @@ export default class ModalWithImage extends Modal {
     super._setEventListeners();
     this._modalImage.addEventListener("submit", (e) => {
```

Comment:
Please, delete the submit handler from `ModalWithImage` because you can't submit this modal